tasks/page/processGerbers.js

```diff
@@ -29,17 +28,21 @@ if (require.main !== module) {
       file = fs.readFileSync(`${repoRootPath}/kitspace.yml`)
     }
     let info = file == null ? {} : yaml.safeLoad(file)
-    const files = globule
-      .find(`${repoRootPath}/**/*`)
-      .map(p => path.relative(repoRootPath, p))
+    let gerberPath = `${repoRootPath}/**/*`
 
     if (info.multi) {
       for (let project in info.multi) {
         if (project === projectPath) {
           info = info.multi[project]
+          gerberPath = `${repoRootPath}/${projectPath}/**/*`
         }
       }
     }
+
+    const files = globule
+      .find(gerberPath)
+      .map(p => path.relative(repoRootPath, p))
+
     const gerbers = gerberFiles(files, info.gerbers).map(p =>
       path.join(repoRootPath, p)
     )
```

Comment:
I think this way it wouldn't allow gerbers to be kept outside of the `projectPath` but we should give that flexibility to people that want to organize their repo in a different way. E.g.

```yaml
multi:
    project_1:
        gerbers: gerbers_1/
    project_2:
        gerbers: gerbers_2/
```

```
.
├── gerbers_1
│   ├── gerber_1.gbr
│   └── gerber_2.gbr
├── gerbers_2
│   ├── gerber_1.gbr
│   └── gerber_2.gbr
├── kitspace.yaml
├── project_1
│   ├── cad_file.cad
│   └── README.md
└── project_2
    ├── cad_file.cad
    └── README.md
```

Reply:
Agreed. Think I got this figured out.